src/common/src/lib.rs

```diff
@@ -82,6 +82,7 @@ pub use {
     risingwave_common_metrics as metrics, risingwave_common_secret as secret,
     risingwave_license as license,
 };
+pub mod cdc;
```

Comment:
I think this should be put in the `connector` crate instead of `common`. 
IIRC `frontend`, `meta`, `stream` both depend on `connector`

Reply:
refactored